include/zim/archive.h

```diff
@@ -51,6 +51,25 @@ namespace zim
    * An `Archive` is read-only, and internal states (as caches) are protected
    * from race-condition. Therefore, all methods of `Archive` are threadsafe.
    *
+   * Zim archives exist with two different namespace schemes: An old one and the new one.
+   * The method `hasNewNamespaceScheme` permit to know which namespace is used by the archive.
+   *
+   * When using old namespace, user entries may be stored in different namespaces (historically `A`, `I`, `J` or `-`).
+   * So path of the entries contains the namespace as a "subdirectory" : `A/foo.html`, `I/image.png`, ...
+   * All API taking or returing a path expect/will return a path with the namespace.
+   *
+   * When using new namespace, user entries are always stored whithout namespace.
+   * (For information, they are stored in the same namespace `C`. But consider there is no namespace as all API masks it)
+   * As there is no namespace, paths don't contain it: `foo.hmtl`, `image.png`, ...
+   * All API taking or returning a path expect/will return a path without namespace.
+   *
+   * This difference may seems complex to handle but no so much.
+   * As all paths returned by API is consistent with paths expected, you simply have to use the path as it is.
+   * Forget about the namespace and if a path has it, simply consider it a as subdirectory.
+   * The only place it could be problematique is when you already have a path stored somewhere (bookmark, ...)
+   * using a scheme and use it on a archive with another scheme. For this case, the method `getEntryByPath`
+   * has a compatibility layer trying to transform a path to the new scheme as a fallback if the entry is not found.
+   *
```

Comment:
1. It's better to replace "subdirectory" with "top level directory".
2. problematique → problematic (there are a few other typos, but it would be too impolite enumerating them individually; I believe you will notice all of them if you proofread this text).